evm/src/memory/columns.rs

```diff
@@ -35,8 +35,11 @@ pub(crate) const CONTEXT_FIRST_CHANGE: usize = VALUE_START + VALUE_LIMBS;
 pub(crate) const SEGMENT_FIRST_CHANGE: usize = CONTEXT_FIRST_CHANGE + 1;
 pub(crate) const VIRTUAL_FIRST_CHANGE: usize = SEGMENT_FIRST_CHANGE + 1;
 
+// Used to zero-initialize (or not) the memory.
+pub(crate) const NEW_FIRST_READ: usize = VIRTUAL_FIRST_CHANGE + 1;
```

Comment:
nit: may deserve a clearer comment / var naming? A bit hard to parse without looking at the rest of the code.